### ISSUE TITLE
Cannot find module "react/lib/ReactTestUtils" - react@15.4.0

### ISSUE THREAD

**Issue by luisrudge** [Legitcode/tests#42]
Since the update, I'm getting the error `Cannot find module "react/lib/ReactTestUtils"`

Probably related to this: https://github.com/facebook/react/issues/8314

**Comment by zackify** [comment]
Did `npm install react-dom --save` fix it?

**Comment by luisrudge** [comment]
I already have react-dom in my app

**Comment by zackify** [comment]
Well nothing in this library pulls from react/lib. So I'm not sure how to help :/

**Comment by luisrudge** [comment]
Yeah.. I searched for that as well, but couldn't find anything. I just posted here to see if someone else was having the same issue. It's probably an out of date dependency

**Comment by 7s4r** [comment]
Try to update your package react-addons-test-utils to latest. I fixed this error by updating from "^15.1.0" to "^15.4.1". And also "react" to "^15.3.2" & "react-dom" to "^15.4.1".

**Comment by activedecay** [comment]
with "react": "15.3.0" ran into this error; it was fixed with "react": "^15.4.1"
```
ERROR in ./~/react-dom/lib/ReactDOMUnknownPropertyHook.js
Module not found: Error: Can't resolve 'react/lib/ReactComponentTreeHook' in '/home/jfaust/Documents/hack/react-boilerplate/node_modules/react-dom/lib'
 @ ./~/react-dom/lib/ReactDOMUnknownPropertyHook.js 15:29-72
```
